tensorboard/BUILD

```diff
@@ -325,6 +342,28 @@ tf_web_library(
     ],
 )
 
+tensorboard_zip_file(
+    name = "dev_webfiles",
+    deps = [":dev_assets"],
+)
+
+tf_web_library(
+    name = "dev_assets",
```

Comment:
I assume the idea is that we'll replace this fairly soon with something more substantially different?  Otherwise, would be good to have a comment saying to keep the srcs/deps in sync with the non-dev `assets`.

Reply:
Yup, #4589 gives you glimpse of what the difference might be.